docs/Developers/Newcomers_Guide.rst

```diff
@@ -97,104 +104,97 @@ Step 2. Grab an Invitation to the Organization
 ----------------------------------------------
 
 Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is your first step towards contributing.
-A maintainer will command ``cobot`` (our gitter bot) to invite
-you and be part of the Newcomer team.
+invitation to our org.
+This is your first step towards contributing.
+A maintainer will command ``cobot`` (our gitter bot) to invite you and be part
+of the Newcomer team.
```

Comment:
if possible could you put those layout changes in a separate commit? Makes it easier to track what has changed semantically^^